### PR TITLE
Fix older compatible functions on the aws-lambda context on the Types 2.0 branch

### PR DESCRIPTION
**Pull request by StefH**
Improvement to existing type definition on the **Types 2.0 branch** : Fix older compatible functions on the aws-lambda context.

For details see : https://github.com/DefinitelyTyped/DefinitelyTyped/pull/11924

### CODE REVIEW

**Comment by vvakame** [comment]
@zhengbli What's mean `Revision needed` in this case?

**Review by zhengbli — CHANGES_REQUESTED** [review]

**Comment by zhengbli** [comment]
I didn't realize the new "code review" feature of github needs an extra "submit" steps ....

File: aws-lambda/index.d.ts [hunk 1]
```diff
@@ -3,65 +3,68 @@
 // Definitions by: James Darbyshire <https://github.com/darbio/aws-lambda-typescript>, Michael Skarum <https://github.com/skarum>, Stef Heyenrath <https://github.com/StefH/DefinitelyTyped>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
+declare namespace AWSLambda {
```
Comment (StefH) [inline aws-lambda/index.d.ts]: Code is changed.

**Comment by StefH** [comment]
Review comments are processed, please check.

**Comment by vvakame** [comment]
conflict occured. could you resolve it?

**Comment by StefH** [comment]
@vvakame very strange that there is a conflict because I'm the only editor for those files.
Or did you or @zhengbli already update my files after code-review?

**Comment by vvakame** [comment]
@zhengbli gentle ping

**Review by zhengbli — APPROVED** [review]

**Comment by zhengbli** [comment]
Thanks @StefH !

**Pull request merged**